nightmare/paths/graphs.spec.js

```diff
@@ -0,0 +1,127 @@
+/* eslint max-len: ["error", 500] */
+import { catchErrors } from 'api/utils/jasmineHelpers';
+import { toMatchImageSnapshot } from 'jest-image-snapshot';
+import createNightmare from '../helpers/nightmare';
+import insertFixtures from '../helpers/insertFixtures';
+import { loginAsAdminAndGoToSettings } from '../helpers/commonTests';
+
+expect.extend({ toMatchImageSnapshot });
+
+const localSelectors = {
+  pagesButton:
+    '#app > div.content > div > div > div.settings-navigation > div > div:nth-child(1) > div.list-group > a:nth-child(5)',
+  createNewPageButton:
+    '#app > div.content > div > div > div.settings-content > div > div.settings-footer > a',
+  savePageButton:
+    '#app > div.content > div > div > div.settings-content > div > form > div.settings-footer > button.save-template',
+  pageTitleInput:
+    '#app > div.content > div > div > div.settings-content > div > form > div.panel.panel-default > div.metadataTemplate-heading.panel-heading > div > div > input',
+  pageContentsInput:
+    '#app > div.content > div > div > div.settings-content > div > form > div.panel.panel-default > div.panel-body.page-viewer.document-viewer > div > div.tab-content.tab-content-visible > textarea',
+  createdPageLink: '.document-viewer > div.alert-info a',
+};
+
+const nightmare = createNightmare();
+
+const graphs = {
+  barChart: '<BarChart property="super_powers" context="58ad7d240d44252fee4e6208" />',
+  pieChart: '<PieChart property="super_powers" context="58ad7d240d44252fee4e6208" />',
+  listChart:
+    '<ListChart property="super_powers" context="58ad7d240d44252fee4e6208" excludeZero="true" />',
+};
+
+describe('pages path', () => {
+  beforeAll(async () => insertFixtures());
+  afterAll(async () => nightmare.end());
+
+  describe('login', () => {
+    it('should log in as admin then click the settings nav button', done => {
+      loginAsAdminAndGoToSettings(nightmare, catchErrors, done);
+    });
+  });
+
+  describe('Graphs in Page', () => {
+    it('should create a basic page', async () => {
+      await nightmare
+        .goto('http://localhost:3000/en/settings')
+        .clickLink('Pages')
+        .clickLink('Add page')
+        .wait('.page-creator')
+        .write(localSelectors.pageTitleInput, 'Page data viz')
+        .write(localSelectors.pageContentsInput, '</p><Dataset />')
+        .clickLink('Save')
+        .wait(1000);
```

Comment:
@eric remove this wait, a way to do it is to wait for the success notification and clicking on it to close it.